lib/openstack_api/openstack_query.py

```diff
@@ -180,6 +180,9 @@ def parse_and_output_table(
         property_funcs = self.get_default_property_funcs(object_type, cloud_account)
         output = self.parse_properties(items, properties_to_select, property_funcs)
 
+        if len(output) == 0:
+            return "No results"
```

Comment:
Return `None` then interpret this in the action as no results. Since the lib layer shouldn't be doing anything user facing (bar exceptions)